src/directLine.ts

```diff
@@ -262,7 +260,7 @@ export class DirectLine implements IBotConnection {
     public activity$: Observable<Activity>;
 
     private domain = "https://directline.botframework.com/v3/directline";
-    private webSocket = true;
+    private webSocket;
```

Comment:
hmmm I don't think this webSocket logic is right. Below you seem to be setting it twice.